samples/graphene-hello/auto-preload/hello.manifest.template

```diff
@@ -0,0 +1,43 @@
+loader.preload = "file:$(GRAPHENEDIR)/Runtime/libsysdb.so"
+
+# run Marblerun premain
+loader.env.LD_PRELOAD = "./premain-graphene.so"
+loader.env.LD_LIBRARY_PATH = "lib:/lib/x86_64-linux-gnu:/usr/lib:/usr/lib/x86_64-linux-gnu"
```

Comment:
For the sake of this demo, properly not all of them. Only those `ldd` outputs to us.

Not sure though if hardcoding every single library makes a good demo if someone wants to use this example to load their own application in there, as it could require more shared libraries than the ones we provide.

Reply:
I don't think that this is how it works in Graphene. All required libs are in the folder mounted at /lib, except premain-graphene.so, but this one has an explicit path (./...). It actually works with just "/lib" on my VM. Can you double-check?

Reply:
Does indeed work (now?). Will reduce it back to /lib.